Source/WebCore/svg/SVGGradientElement.cpp

```diff
@@ -3,6 +3,7 @@
  * Copyright (C) 2004, 2005, 2006, 2007 Rob Buis <buis@kde.org>
  * Copyright (C) Research In Motion Limited 2010. All rights reserved.
  * Copyright (C) 2018-2019 Apple Inc. All rights reserved.
+ * Copyright (C) 2023 Igalia S.L.
```

Comment:
I guess we have to include 2024 these days :)

Reply:
That code is actually from 2023, but I will remember to add 2024, where appropriate.

Reply:
Done, selectively.